### PR TITLE
added image styling

### PR DESCRIPTION
**Pull request by benfes**
**What is the purpose of this pull request?**
Added image components to the library

**Which issue number does this fix?**
https://github.com/cpdevelops/simple.css/issues/4

### CODE REVIEW

**Comment by cpdevelops** [comment]
Hey there! Before continuing with this pull request, please make sure you have read through the README and CONTRIBUTING files, and have added your changes to the demo site at index.html.

File: dist/simple.css [hunk 1]
```diff
@@ -1,9 +1,10 @@
-/*
- * Welcome to simple.css
- * Simple.css is a CSS framework designed to be simple to use and edit
- *
- * Designed by Colin Pokowitz (cpdev) - Written in 2017
- * MIT Licence
+/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */
+/* Document
```
Comment (cpdevelops) [inline dist/simple.css]: Why did you delete the previous information? This should stay, and should not be replaced by anything.

File: .idea/encodings.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (cpdevelops) [inline .idea/encodings.xml]: What is the purpose of these XML files?

File: src/styles/simple.scss [hunk 3]
```diff
@@ -17,4 +17,4 @@
 @import './components/header';
 @import './components/list';
 @import './components/alert';
-@import './components/image';
+@import './components/code';
```
Comment (cpdevelops) [inline src/styles/simple.scss]: You also seem to have removed your own image styling with this merge, along with the other issues.

**Comment by cpdevelops** [comment]
I am sorry, but with all these issues I will have to go ahead and close this PR. If you decide to reopen this, please make sure that you have fixed all the issues mentioned above and provide us with a working component that follows the current format.

File: .idea/encodings.xml (hunk 2 above)
Comment (dvlden) [reply .idea/encodings.xml]: Since you are using one of `JetBrains` IDE's for development, make sure to add `.idea` into `.gitignore`

Comment (cpdevelops) [reply .idea/encodings.xml]: @dvlden I will go ahead and add this to masters ```.gitignore```
